frontend/src/components/TheHeader.vue

```diff
@@ -2,31 +2,36 @@
   <div class="p-2 items-center bg-slate-800 text-white">
     <div class="inline-flex flex-row items-center">
       <a href="https://bytebase.com?ref=dbcost" target="_blank">
-        <img class="h-6" src="../assets/icon/dbcost.png" alt="Bytebase" />
+        <img
+          class="h-8"
+          src="../assets/icon/dbcost-logo-full.png"
+          alt="Bytebase"
+        />
       </a>
-      <span class="inline font-bold text-xl text-center">DB Cost</span>
     </div>
 
     <div class="mt-1.5 inline-flex float-right items-center space-x-2">
       <div v-is="'script'" async src="https://buttons.github.io/buttons.js" />
-      <a
-        class="github-button"
-        href="https://github.com/bytebase/dbcost"
-        data-color-scheme="no-preference: light; light: light; dark: dark;"
-        data-show-count="false"
-        aria-label="Star bytebase/dbcost on GitHub"
-        >Star</a
-      >
+      <div class="mt-1">
+        <a
+          class="github-button"
+          href="https://github.com/bytebase/dbcost"
+          data-color-scheme="no-preference: light; light: light; dark: dark;"
+          data-show-count="false"
+          aria-label="Star bytebase/dbcost on GitHub"
+          >Star</a
+        >
+      </div>
 
-      <div class="flex items-center -mt-1.5">
+      <div class="-mt-0.5 flex items-center">
```

Comment:
```suggestion
      <div class="mt-0.5 flex items-center">
```

Reply:
Is this a typo?

Reply:
This is intentional. Otherwise the icon doesn't align with the github button